src/cmd/cmd.cpp

```diff
@@ -311,7 +311,7 @@ int main(int argc, char **argv)
     qputenv("OPENSSL_CONF", opensslConf.toLocal8Bit());
 #endif
 
-    qsrand(QTime::currentTime().msec() * QCoreApplication::applicationPid());
+    qsrand(QDateTime::currentMSecsSinceEpoch());
```

Comment:
Why did you remove this? I remember this was be added for smashbox doing parallel testing which had before this lead to duplicate transferids used in chunking